client/src/utils/GameContext.tsx

```diff
@@ -36,10 +37,13 @@ export const GameProvider = ({ children }: Props) => {
     }
   }
 
-  const joinGameRoom = async (playerID: string) => {
+  const joinGameRoom = async (playerID: string, isRobotMode: boolean) => {
+    const robotMode = isRobotMode ? "enabled" : "disabled"
```

Comment:
Switching from boolean to string. Boolean false converts to None on the server side;